docs-src/stargate-core/modules/ROOT/pages/FAQ.adoc

```diff
@@ -9,31 +9,32 @@ https://github.com/stargate/stargate[Stargate repo] and use the bug label.
 It's also helpful to include the full stack trace and the Stargate and database
 version that you are using so that we can efficiently debug.
 - If you've found an issue with the documentation, please file an issue in the
-https://github.com/stargate/docs[Stargate docs repo]. Pull requests are also
+https://github.com/stargate/docs[Stargate docs repo, window="_blank"]. Pull requests are also
```

Comment:
Should we add `, window="_blank"` to Stargate repo above as well? just to be consistent?

Reply:
Yes, good idea. Will do in next commit.